test/parallel/test-buffer-alloc.js

```diff
@@ -1,15 +1,11 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var Buffer = require('buffer').Buffer;
+const Buffer = require('buffer').Buffer;
+const SlowBuffer = require('buffer').SlowBuffer;
 
-// counter to ensure unique value is always copied
-var cntr = 0;
-
-var b = Buffer.allocUnsafe(1024);
-
-console.log('b.length == %d', b.length);
+const b = Buffer.allocUnsafe(1024);
 assert.strictEqual(1024, b.length);
```

Comment:
The API for `assert.strictEqual()` and friends indicates that the actual value received should be the first parameter and the expected value should be second but it's backwards here and below. Feel free to leave it as is (since this isn't code you wrote, it's just an adjacent line), but if you want to swap them, hey, go for it.